### ISSUE TITLE
Problem with large .wav files ( bigger then 500MB)

### ISSUE THREAD

**Issue by mitkostankov** [datacompboy/WavPlayer#11]
When the WavPlayer process *.wav file which is bigger then 500MB, the memory overflows and the flash plugin crashes.

**Comment by livem** [comment]
so big file ......

**Comment by datacompboy** [comment]
Yes, known problem. Currently file fully buffers in RAM for rewind.
It should be possible to limit maximum size to some amount of RAM by changing File() class to keep only +- some amount of samples around current point.
I have no time to invest into that, if anyone have similar problem -- i'll be glad to merge change.
